docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -19,79 +19,118 @@ This guide will walk you through the steps of doing this migration.
 
 ---
 
-## Prerequisites
+## Step 1: Take inventory of your existing Airflow usage
 
-To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+Making migrations is complicated, there are lots of moving pieces and it will be important for you to go in knowing exactly how much scope and complexity you'll be tackling.
+
+There are a few things you'll want to do ahead of time, specifically you'll want to make a note of:
```

Comment:
@Ramshackle-Jamathon I was thinking something in this tone:

### How easy is this going to be, really?

You may be coming to this document/library in a skeptical frame of mind, having previously been burned by projects that claim to have 100% foolproof, automated migration tools. We want to assure you that we are _not_ making that claim.

The `dagster-airflow` migration library should be viewed as a powerful _accelerant_ of migration, rather than guaranteeing completely _automated_ migration. The amount of work required is proportional to the complexity of your installation. Smaller implementations of less complexity can be trivial to migrate, making it appear virtually automatic; larger, more complicated, or customized implementations require more investment. 

For larger installations, teams that already adopt devops practices and/or have standardized usage of Airflow will have a smoother transition. 

Some concrete examples:

* If you rely on the usage of the Airflow UI to set production connections or variables, this will require a change of workflow as you will no longer have the Airflow UI at the end of this migration. If instead you rely on code-as-infrastructure patterns to set connections or variables, migration more straightforward. 
* If you have standardized on a small set or even a single operator type (e.g. the K8sPodOperator), migration will be easiser. If you use a large number of operator types with a wide range of infrastructure requirements, migration will be more work.
* If you dynamically generate your Airflow DAGs from a higher-level API or DSL (e.g. yaml), the migration will be more straightforward than all your stakeholders directly creating Airflow DAGs.

Even in the case that requires some infrastructure investment, `dagster-airflow` dramatically eases migration, by orders of magnitude. The cost can be borne by a single or small set of infrastructure-oriented engineers, and it dramatically reduces coordination costs, as you do not have to move all of your stakeholders over to new APIs simultaneously. In our experience, practitioners welcome the change, because of the immediate improvement in tooling, stability, and development speed.

Reply:
ah, do you mind if I lift this directly into the docs?